.github/workflows/live_artifacts.yml

```diff
@@ -80,7 +80,8 @@ jobs:
         env:
           COMPILER: gcc
         run: ci/actions/linux/install_deps.sh
-      - name: Deploy Docker (nanocurrency/nano)
+      - name: Deploy Docker Hub (nanocurrency/nano)
+        if: ${{ github.repository == 'nanocurrency/nano-node' }}
         run: TRAVIS_TAG=${TAG} ci/actions/linux/deploy-docker.sh
         env:
           DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
```

Comment:
not related to your change, but if you can fix the typo at line 94, it is missing closing the parentheses.
`      - name: Deploy Docker (ghcr.io`